src/Console/CacheItemPoolFactoryInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Contentful\Console;
+
+use Psr\Cache\CacheItemPoolInterface;
+
+interface CacheItemPoolFactoryInterface
+{
+    public function __construct();
+
+    /**
+     * @return CacheItemPoolInterface
+     */
+    public function getCacheItemPool();
+}
```

Comment:
Even though I understand you reasoning for having this interface in this namespace, it would be a slight architectural problem, could you please move it to `Contentful\Delivery\Cache`? Thanks!